### ISSUE TITLE
'["China", "France"]' or ['China', 'France'] in docs?

### ISSUE THREAD

**Issue by awalias** [supabase/supabase#22]
![image](https://user-images.githubusercontent.com/458736/74817677-43dc3380-5338-11ea-8a85-071a9fd8e20d.png)

https://supabase.io/docs/library/get/#in

`'["China", "France"]'` or `['China', 'France']` ?
need to test whether this should be wrapped as a string or not

currently facing other issues which means I can't test it right now, so saving here for later investigation

**Comment by kiwicopple** [comment]
> need to test whether this should be wrapped as a string or not

Should be a plain array: 
https://github.com/supabase/postgrest-js/blob/3eca6e8459630bd959355fad1b48e5ee31b33bf7/src/utils/Filters.js#L158

Not sure why it's failing, tests seem to be OK: 
https://github.com/supabase/postgrest-js/blob/master/README.md#status

**Comment by awalias** [comment]
this is the error

```javascript
      text: '{"details":"unexpected \\"2\\" expecting \\"(\\"","message":"\\"failed to parse filter (in.2,3)\\" (line 1, column 4)"}',
      method: 'GET',
      path: '/rest/v1/users?apikey=examplekey&id=in.2,3&select=%2A'
```

**Comment by awalias** [comment]
ok so looks like it should be getting wrapped with parenthesis like:
`http://localhost:8000/rest/v1/users?apikey=examplekey&id=in.(2,3)&select=*`

**Comment by kiwicopple** [comment]
Fixed in postgrest-js@0.8.0
